test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -371,7 +433,12 @@ public void CreateDisplayMetadata_DisplayAttribute_LocalizeProperties()
                 .Setup(f => f.Create(It.IsAny<Type>()))
                 .Returns(stringLocalizer.Object);
 
-            var provider = new DataAnnotationsMetadataProvider(stringLocalizerFactory.Object);
+            var options = new TestOptionsManager<MvcDataAnnotationsLocalizationOptions>();
+            new MvcDataAnnotationsLocalizationOptionsSetup().Configure(options.Value);
```

Comment:
Since this isn't a test of `MvcDataAnnotationsLocalizationOptionsSetup`, use the same setup as https://github.com/aspnet/Mvc/pull/5287/files#diff-a39e8ac83ba85b7a9293e1081040b339R187